autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml

```diff
@@ -23,19 +24,28 @@
       object_check_shiftable_ratio: 0.6           # [-]
       object_check_min_road_shoulder_width: 0.5   # [m]
 
+      # For safety check
+      safety_check_backward_distance: 50.0        # [m]
+      safety_check_time_horizon: 10.0             # [s]
+      safety_check_idling_time: 1.5               # [s]
+      safety_check_accel_for_rss: 2.5             # [m/ss]
+
       # For lateral margin
       object_envelope_buffer: 0.3                 # [m]
       lateral_collision_margin: 1.0               # [m]
       lateral_collision_safety_buffer: 0.7        # [m]
 
+      # For longitudinal margin
+      longitudinal_collision_margin_buffer: 0.0   # [m]
+
       prepare_time: 2.0                           # [s]
       min_prepare_distance: 1.0                   # [m]
       min_avoidance_distance: 10.0                # [m]
 
       min_nominal_avoidance_speed: 7.0  # [m/s]
       min_sharp_avoidance_speed: 1.0    # [m/s]
 
-      road_shoulder_safety_margin: 0.5 # [m]
+      road_shoulder_safety_margin: 0.0 # [m]
```

Comment:
@satoshi-ota which one better to use?
original PR don't change this parameter but the value is 0.0
https://github.com/autowarefoundation/autoware.universe/pull/2534/files

Reply:
I think 0.5 is better as default.

Reply:
ok :+1:

Reply:
[630a29b](https://github.com/autowarefoundation/autoware_launch/pull/145/commits/630a29bea645fe2d43d462fbec1c8e0e8ff4ce29)